### PR TITLE
Documentation updates

### PR DESCRIPTION
**Pull request by varunarora**
Just making the initial work visible, lots more to come

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,45 +1,73 @@
...
+## How it works
+
+- A [design document of the underlying ideas](https://github.com/PaddlePaddle/Paddle/blob/develop/doc/fluid/design/onnx/onnx_convertor.md) behind how this program converts the model from PaddlePaddle to ONNX.
+- Understand [PaddlePaddle's (non-)graph way of representing a deep learning program](https://github.com/PaddlePaddle/Paddle/blob/develop/doc/fluid/design/concepts/program.md), a `ProgramDesc`.
+
+
+## Status
 
-## Testing
+Targets Paddle->ONNX conversion for now (or what's called **frontend** in the ONNX world), and will consequently support the reverse too.
```
Comment (sidgoyal78) [inline README.md]: Maybe targets -> "The current release targets"?

**Review by sidgoyal78 — APPROVED** [review]
LGTM, just a minor paraphrasing suggestion.

File: README.md [hunk 2]
```diff
@@ -1,45 +1,73 @@
...
+
+## Testing / validation
+
+To validate the similarity of the PaddlePaddle Fluid model and the exported ONNX model, run the following:
+
+```
+python validate.py --fluid_model <path_to_paddlepaddle_fluid.model> --onnx_model <path_to_exported_model.onnx>
+```
+
+This validation aims for an output tensor comparison precision at 5-decimal places.
```
Comment (kuke) [inline README.md]: -> at 5-decimal places by default. About more arguments, please run 
```
python validate.py --help
```

File: README.md [hunk 3]
```diff
@@ -51,5 +79,23 @@ We aim to at least support all the models from our model bank. During our prelim
...
+
+
+## Contribute
+
+In the highest spirits of open-source, we welcome your contributions! The philosophy around contribution at this stage is to get **better model coverage**. Some of the code for training popular models is already written for PaddlePaddle Fluid. For code not written for PaddlePaddle Fluid, we invite writing this training code first (and possibly contributing it to the `tests` inside PaddlePaddle's main repo) and then:
+
+- Read the documents in the `How it works` section above.
+- (Optional) Create an issue sharing need for support for a new model. Assign to yourself.
+- Add the necessary operator conversion logic in `fluid_onnx/ops.py`.
+- Write tests for the newly introduce operator convertors.
```
Comment (kuke) [inline README.md]: `introduced operator conversion function`

**Review by kuke — COMMENTED** [review]
Almost LGTM

**Pull request merged**